### PR TITLE
[zh-cn] resync: volumes.md & download.md

### PR DESCRIPTION
**Pull request by hs0210**
Signed-off-by: Hu Shuai <hus.fnst@fujitsu.com>

### CODE REVIEW

File: content/zh-cn/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -533,11 +533,12 @@ keyed with `log_level`.
...
-A `downwardAPI` volume is used to make downward API data available to applications.
-It mounts a directory and writes the requested data in plain text files.
+A `downwardAPI` volume makes {{< glossary_tooltip term_id="downward-api" text="downward API" >}}
+data available to applications. Within the volume, you can find the exposed
+data as read-only files in plain text format.
 -->
-`downwardAPI` 卷用于使 downward API 数据对应用程序可用。
-这种卷类型挂载一个目录并在纯文本文件中写入所请求的数据。
+`downwardAPI` 卷用于使 {{< glossary_tooltip term_id="downward-api" text="downward API" >}}
+数据对应用程序可用。在该卷中，挂载数据以纯文本格式的只读文件形式存在。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/storage/volumes.md:541]:
```suggestion
`downwardAPI` 卷用于为应用提供 {{< glossary_tooltip term_id="downward-api" text="downward API" >}} 数据。
在这类卷中，所公开的数据以纯文本格式的只读文件形式存在。
```

**Comment by hs0210** [comment]
> The content/zh-cn/docs/concepts/storage/volumes.md file have some other un-synced places, please check with local diff tool.

@Sea-n Done. PTAL.

**Review by ydFu — APPROVED** [review]
/lgtm
LGTM,Thank for sync.

**Pull request merged**